### PR TITLE
surf: 2.0 -> unstable-2019-02-08

### LINKED ISSUE #81629: Update surf package
Prefer Git revision rather release.
Version 2.0 is from 2017-03-28.

- https://surf.suckless.org
- https://github.com/NixOS/nixpkgs/tree/master/pkgs/applications/networking/browsers/surf

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Motivation for this change

Closes #81629

@alexherbo2 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by alexherbo2** [comment]
@SuperSandro2000 Can you update the description to replace `Surf` → `surf` and `WebKit/GTK` → `WebKitGTK`?

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 Can you update the description to replace `Surf` → `surf` and `WebKit/GTK` → `WebKitGTK`?

Changed that.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101430` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>surf (surf-webkit2)</li>
    <li>surf-display</li>
  </ul>
</details>

File: pkgs/applications/networking/browsers/surf/default.nix [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-, glib, glib-networking, gsettings-desktop-schemas, gtk, libsoup, webkitgtk
+, glib, gcr, glib-networking, gsettings-desktop-schemas, gtk, libsoup, webkitgtk
 , xorg, dmenu, findutils, gnused, coreutils
 , patches ? null
 }:
 
 stdenv.mkDerivation rec {
   pname = "surf";
-  version = "2.0";
+  version = "2019-02-08-unstable";
```
Comment (sikmir) [inline pkgs/applications/networking/browsers/surf/default.nix:10]:
```suggestion
  version = "unstable-2019-02-08";
```
See https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101430` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>surf (surf-webkit2)</li>
    <li>surf-display</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101430` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>surf</li>
  </ul>
</details>

**Pull request merged**